src/utils/media/effects/virtual-background/WebGLCompositor.js

```diff
@@ -536,6 +536,44 @@ export default class WebGLCompositor {
 		}
 	}
 
+	/**
+	 * Render the source video as is, without applying any kind of effects
+	 *
+	 * @param {number} outW - Output width.
+	 * @param {number} outH - Output height.
+	 */
+	_renderWithoutEffects(outW, outH) {
+		const gl = this.gl
+
+		gl.bindFramebuffer(gl.FRAMEBUFFER, null)
+		gl.viewport(0, 0, outW, outH)
+		gl.clearColor(0, 0, 0, 1)
+		gl.clear(gl.COLOR_BUFFER_BIT)
+
+		// Reuse the blend program but disable mask/background sampling
+		gl.useProgram(this.progBlend)
+		this._setupVertexAttributes(this.progBlend)
+
+		gl.uniform1i(gl.getUniformLocation(this.progBlend, 'u_inputFrame'), 0)
+		gl.uniform1i(gl.getUniformLocation(this.progBlend, 'u_personMask'), 1)
+		gl.uniform1i(gl.getUniformLocation(this.progBlend, 'u_blurredFrame'), 2)
+		gl.uniform1i(gl.getUniformLocation(this.progBlend, 'u_background'), 3)
+		gl.uniform2f(gl.getUniformLocation(this.progBlend, 'u_coverage'), 0.0, 0.0)
+		gl.uniform1f(gl.getUniformLocation(this.progBlend, 'u_lightWrapping'), 0.0)
+		gl.uniform1i(gl.getUniformLocation(this.progBlend, 'u_mode'), 2) // custom dummy mode
```

Comment:
shouldn't `u_mode` be -1 to match the other code (or we can change -1 to 2 in other places as well?)